### DISCUSSION TITLE
Can I contribute a Chinese translation for docs?

### DISCUSSION THREAD

**Discussion by zhtyyx** [dagster-io/dagster#4535] [General]
Is it needed?

**Comment by yuhan** [comment]
Hi @zhtyyx! Thanks for reaching out and apologies for missing this discussion.
We currently don't have a concrete plan for translating docs, but it is something we've discussed internally -- I believe there are indeed needs but we currently don't have a good understanding of them, and we'd love to explore and understand! 

If you're still interested (sorry I know it's been a while), please email me at my Github handle at elementl.com :) I'm happy to connect and learn more about your needs to the docs as well as your interests of contributing to the project.

**Reply by zhtyyx** [reply]
Hi @yuhan! 
Thanks for the reply.
I'm still interested in contributing to the project, and willing to translate docs when needed.
I'm looking forward to hearing from you.
